### ISSUE TITLE
CSV output generates error

### ISSUE THREAD

**Issue by EyalLavi** [ebu/benchmarkstt#68]
CSV is an output option (`-o`) in the documentation (https://ebu.github.io/benchmarkstt/cli.html) but it generates an error. Personally I'm happy to remove CSV from the documentation rather than fix this.

```
benchmarkstt  Item2.ref.txt Item2.hyp.txt --wer -o csv
usage: benchmarkstt [-rt REFERENCE_TYPE] [-ht HYPOTHESIS_TYPE]
                    [-o {json,markdown,restructuredtext}]
                    [--diffcounts [differ_class]] [--wer [mode]
                    [differ_class]] [--worddiffs [dialect] [differ_class]]
                    [--config file [section] [encoding]] [--lowercase]
                    [--regex search [replace]] [--replace search [replace]]
                    [--replacewords search replace] [--unidecode] [--log]
                    [--version]
                    [--log-level {critical,fatal,error,warn,warning,info,debug,notset}]
                    [--help]
                    reference hypothesis
benchmarkstt: error: argument -o/--output-format: invalid choice: 'csv' (choose from 'json', 'markdown', 'restructuredtext')
(venv) lavie01@mc-n354732:~/code/git/benchmarkstt/venv$ 
```

**Comment by EyalLavi** [comment]
After checking with @danielthepope this is not a bug, but we need to remove the CSV from the documentation and help.

**Comment by MikeSmithEU** [comment]
Documentation has been updated to newest version, needed to fix documentation tho (see #81), not tagged as new version (no functional changes).
